fcrepo-webapp/src/main/webapp/static/constraints/ContainerConstraints.rdf

```diff
@@ -30,5 +30,11 @@
         <rdfs:comment xml:lang="en">Properties in the &lt;http://fedora.info/definitions/v4/repository#&gt;
           namespace, and types in the &lt;http://fedora.info/definitions/v4/repository#&gt; and
           &lt;http://www.w3.org/ns/ldp#&gt; namespace can only be modified by the repository.</rdfs:comment>
+        <rdfs:comment xml:lang="en">
+			Fedora’s versioning interaction model implements Memento. This interaction model is independent
+			of any other interaction model and MUST be declared at the time of resource creation by using a
```

Comment:
Note: This is coming down the line: https://github.com/fcrepo/fcrepo-specification/pull/397